definitions/horizontal_scaling.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Horizontal Scaling
+status: Feedback Appreciated
+category: Concept
+---
+
+## Horizontal Scaling
+
+### What it is
+
+In [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) perspective Horizontal Scaling is type of [scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) whose approach is to add additional instances instead of moving into a larger instance size.
```

Comment:
This is great. A few points:

- Not everyone may be familiar with what instance means in this context. Node is a term that has already been captured in the glossary, so, if we use node instead of instance, people can easily look it up
- You say "adding additional instances," but to what? The subject of the sentence is missing 
-  Since the example refers to the sentence before, it should be part of the same paragraph 
- Horizontal scaling is a qualified noun, not a name, and should not be capitalized

Here's my suggestion: 

Horizontal scaling is a scaling technique where a system's capacity is increased by adding more [nodes](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) versus adding more compute resources to individual nodes (the latter is referred as vertical scaling). For example, ...